### ISSUE TITLE
Remove commented code in CSS files

### ISSUE THREAD

**Issue by heliomar-pena** [ildebr/ild_portfolio#1]
In Land.scss file you have some blocks of code commented. It's recommended to remove it instead since in the future it is so possible that you miss why you commented on it. But it will look important (there should be any reason why you commented on it, is not?), so you will not want to delete it, but also will not be able to use it because you don't remember what it does. So it's better to remove it!. Anyway, you have the git history if you need it

https://github.com/ildebr/ild_portfolio/blob/master/src/pages/Land/Land.scss#L84-L92

https://github.com/ildebr/ild_portfolio/blob/master/src/pages/Land/Land.scss#L167-L169

**Comment by ildebr** [comment]
Yes! I was trying both styles and was unsure which one to keep for the scroll bar, but I like the one I have in place so I will remove those comments in a furute push! Thanks!

**Comment by heliomar-pena** [comment]
Great. thanks!.

BTW: I took a look at your portfolio and looks quite attractive. Good work!
